lib/task_helpers/exports.rb

```diff
@@ -13,6 +13,15 @@ def self.parse_options
         opt :all, 'Export read-only objects', :type => :boolean, :default => false
       end
 
+      def ensure_absolute_path(dir)
+        abs_path = File.absolute_path(dir)
+        unless abs_path == dir
```

Comment:
I don't think you need this check... Replacing a value with the same value doesn't hurt anything, and won't be expensive.

Reply:
Thanks, will correct.